src/packages/end-to-end/src/__tests__/ui/auth/create-user.test.ts

```diff
@@ -0,0 +1,52 @@
+import { test, expect } from '@playwright/test';
+import { config } from '../../../config';
+import { randomUUID } from 'crypto';
+
+test('should allow an admin to create a user', async ({ page }) => {
+	const username = randomUUID();
+	await page.goto(config.adminUiUrl);
+	await page.getByPlaceholder('Username').click();
+	await page.getByPlaceholder('Username').fill('darth');
+	await page.getByPlaceholder('Password').click();
+	await page.getByPlaceholder('Password').fill('deathstar123');
+	await page.getByPlaceholder('Password').press('Enter');
+	await page.getByRole('link', { name: 'mikro-orm-my-sql' }).click();
+	await page.getByRole('link', { name: 'Credential' }).click();
+	await page.getByRole('button', { name: 'Create New Credential' }).click();
+	await page.getByLabel('username').click();
+	await page.getByLabel('username').fill(username);
+	await page.getByPlaceholder('Password').click();
+	await page.getByPlaceholder('Password').fill('test1234');
+	await page.getByPlaceholder('Confirm').click();
+	await page.getByPlaceholder('Confirm').fill('test1234');
+	await page.getByRole('button', { name: 'Save' }).click();
+	const element = await page.getByText('has been successfully created');
+	await expect(element).toHaveCount(1);
+
+	// Check that the created item is in the table
+	const tableElement = await page.getByRole('gridcell', { name: username });
+	await expect(tableElement).toHaveCount(1);
+});
+
+test('should not allow a non-admin to create a user', async ({ page }) => {
```

Comment:
It would be slightly nicer if the UI was able to ascertain the user's permission level on a particular entity, and thus show/hide the create/update controls (rather than allowing them to try then fail). But that could be a separate ticket.

Reply:
Yeah we have a ticket already to track this 👍